specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2018-06-01-privatepreview/mysql.json

```diff
@@ -2832,6 +3004,68 @@
         }
       },
       "description": "An error response from the Batch service."
+    },
+     "ServerAdministratorProperties": {
+      "properties": {
+        "administratorType": {
```

Comment:
Same as above concerning consideration of a polymorphic schema

Reply:
Discussed with the team and there will not be any other type in future.  So this is a won't fix